openshift/ci-operator/build-image/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG OCP_VERSION=4.16
```

Comment:
When/how is this file used?

Reply:
This file is used by Prow (Openshift CI) to run our test. It is analog to the "runs-on: ubuntu-latest" on gitactions.

[Here](https://github.com/openshift/release/pull/51325/files#diff-ea4238468a197a400dd9812ad0f0865982a9343ef66b90955a20f5aa8d0dcae9R8) is where we tell Openshift to use this image as base image for our tests

[Here](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/50201/rehearse-50201-pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test/1783149331373625344#1:build-log.txt%3A10) a sample execution where this is image is built.

So after image is built and a container is launch.. Openshift CI pulls func midstream source code there and executes these [steps](https://github.com/openshift/release/pull/51325/files#diff-ea4238468a197a400dd9812ad0f0865982a9343ef66b90955a20f5aa8d0dcae9R39) from the source folder.